src/session/output.rs

```diff
@@ -5,13 +5,13 @@ use std::{
 
 use crate::Value;
 
-pub struct SessionOutputs<'s> {
```

Comment:
Removing the lifetime here means `SessionOutputs` is not bound to the lifetime of the session, meaning it could be accessed after the session is dropped, causing a segfault. Maybe we could add something like `session_bind: PhantomData<&'session ()>` here?

Reply:
If the `Value` all have an `Arc<Session>` then the session will be kept alive until all the `Values` are dropped?